sig-network/charter.md

```diff
@@ -0,0 +1,74 @@
+# SIG Network Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+SIG Network is responsible for the components, interfaces, and APIs which expose networking capabilities to Kubernetes
+users and workloads. SIG Network also provides some reference implementations of these APIs, for
+example kube-proxy as a reference implementation of the Service API.
+
+### In scope
+
+The following topics fall under ownership of this SIG:
+
+- Networking control plane and data paths.
+- Network service abstractions.
+- Service discovery (DNS).
+- Service load balancing (L4, L7).
+- Network security and identity.
+- Cluster connectivity.
+- Cross-cutting concerns such as scalability.
+- Metrics and monitoring associated with networking components.
+
```

Comment:
Do you see the comment from @bgrant0607 in network service mesh discussion? which of the items above would the work there fall under? https://github.com/kubernetes/community/pull/2372#issuecomment-404721484

Reply:
I believe it would fall under some combination of "network service abstractions" and "data paths".

As @thockin mentioned in that thread, there are some aspects of that discussion which span multiple SIGs as well, but I think those two cover the parts relevant to SIG network.

Reply:
Thanks @caseydavenport